### PR TITLE
Article about three Ruby gems that are awesome for judging Ruby code quality

### PR DESCRIPTION
**Pull request by etagwerker**
Hey all, 

This is a new article about judging code quality, not just for Rails, but more for Ruby projects. 

There is a companion page for this article and it is over here: https://fastruby.github.io/quality/

Please check it out. 

Thanks!

### CODE REVIEW

File: _posts/2019-09-29-code-quality-ruby-gems.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+With this report you can get a quick glimpse about the most common code smells
+in your project.
+
+If you check out the Flog section, you will find the methods that are hardest to
+test, the ones that are most complex:
+
+<img src="/blog/assets/images/quality/metric-fu-flog.png" alt="MetricFu: Flog report">
+
+MetricFu is definitely more ambitious than Attractor and RubyCritic, but it hasn't
+been actively maintained for years.
```
Comment (ercohen14) [inline _posts/2019-09-29-code-quality-ruby-gems.md]:
```suggestion
been actively maintained in years.
```

File: _posts/2019-09-29-code-quality-ruby-gems.md [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+[Flay](https://github.com/seattlerb/flay),
+and [Flog](https://github.com/seattlerb/flog) to provide a quality report of
+your Ruby code.
+
+One of the best things about this tool is that it provides a quick overview of
+the project you're analyzing:
+
+<img src="/blog/assets/images/quality/ruby-critic-overview.png" alt="RubyCritic Overview for alphagov/e-petitions">
+
+With this you can get a quick glimpse of the complexity and churn in all files.
```
Comment (ercohen14) [inline _posts/2019-09-29-code-quality-ruby-gems.md]:
```suggestion
With this you can get a glimpse of the complexity and churn in all files.
```

File: _posts/2019-09-29-code-quality-ruby-gems.md [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+
+<img src="/blog/assets/images/quality/metric-fu-flog.png" alt="MetricFu: Flog report">
+
+MetricFu is definitely more ambitious than Attractor and RubyCritic, but it hasn't
+been actively maintained for years.
+
+You can play around with a sample report over here:
+[https://fastruby.github.io/quality/#metric-fu](https://fastruby.github.io/quality/#metric-fu)
+
+## Attractor
```
Comment (ercohen14) [inline _posts/2019-09-29-code-quality-ruby-gems.md]: I would add a link to the title or somewhere in the copy for Attractor, there is no link to it

Comment (etagwerker) [reply _posts/2019-09-29-code-quality-ruby-gems.md]: @ercohen14 done 👍

**Pull request merged**